app/src/main/java/fr/free/nrw/commons/upload/UploadActivity.java

```diff
@@ -480,6 +547,9 @@ private void configureNavigationButtons() {
                 return;
             }
             setTitleAndDescriptions();
+            if (s_count > 3) {
```

Comment:
What is the purpose of `s_count > 3` condition? Lets not use [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)?oldformat=true) in code.

Reply:
s_count > 3 is used to check the condition of multiple uploads. I'll try to figure out an alternative.